concepts/60 Themes and Styles/30 Icons/10 Built-In Icon Library.md

```diff
@@ -1,4 +1,4 @@
-DevExtreme comes with an icon library that provides font icons for all DevExtreme themes. You can use the icons [in widgets](/concepts/60%20Themes%20and%20Styles/30%20Icons/20%20Icons%20in%20Widgets.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Widgets') and [in other page elements](/concepts/60%20Themes%20and%20Styles/30%20Icons/30%20Icons%20in%20Other%20HTML%20Elements.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Other_HTML_Elements') as they are or [customize them](/concepts/60%20Themes%20and%20Styles/30%20Icons/35%20Customize%20Icons.md '/Documentation/Guide/Themes_and_Styles/Icons/#Customize_Icons').
+DevExtreme comes with an icon library that contains SVG and font icons for all DevExtreme themes. You can use the icons [in widgets](/concepts/60%20Themes%20and%20Styles/30%20Icons/20%20Icons%20in%20Widgets.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Widgets') and [in other page elements](/concepts/60%20Themes%20and%20Styles/30%20Icons/30%20Icons%20in%20Other%20HTML%20Elements.md '/Documentation/Guide/Themes_and_Styles/Icons/#Icons_in_Other_HTML_Elements') as they are or [customize them](/concepts/60%20Themes%20and%20Styles/30%20Icons/35%20Customize%20Icons.md '/Documentation/Guide/Themes_and_Styles/Icons/#Customize_Icons').
 
 Available icons are presented below:
```

Comment:
I'd like to revise this sentence without using "presented". Can we say the following instead?

```suggestion
The following icons are available:
```